frontend/pulse3d/src/components/uploadForm/AnalysisParamForm.js

```diff
@@ -154,7 +154,13 @@ export default function AnalysisParamForm({
 
   const pulse3dVersionGte = (version) => {
     const { selectedPulse3dVersion } = inputVals;
-    return selectedPulse3dVersion && semverGte(selectedPulse3dVersion, version);
+
+    // this currently gets called on render when selectedPulse3dVersion is undefined
+    const versionNameOnly = selectedPulse3dVersion
```

Comment:
Is there anyway the "testing" label could be just applied to the display names in the dropdown to allow the versions to be stored just as semvers? That would make it so you don't have to split the tag off everywhere you need to use the actual version

Reply:
Yeah I was thinking about a better way to handle it.. but it will need to be handled either way.. if we just add it to the dropdown options, we'd need to store an  array of just semver versions and then an array with the states, then constantly switch between the two during selections. I'll give it a go